src/service/FileTransferService.ts

```diff
@@ -46,12 +46,6 @@ export class FileTransferService {
                 if (response.status === CREATED && response.data.id) {
                     return response.data.id;
                 }
-            }).catch((err) => {
```

Comment:
Why is that removed?

Reply:
Please bring that back. We do not want to expose errors from HTTP library outside.

Reply:
i've put this code back. We are also catching these errors in the controller so that we can render a suitable error on the page for unsupported mime type